src/Daml.Ledger.Client/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
```

Comment:
Can you remove the commented-out lines?

Reply:
ok, it's a visual studio generated file - generates all those comments!

Reply:
done